server/jetstream_api.go

```diff
@@ -505,9 +505,9 @@ const JSApiConsumerListResponseType = "io.nats.jetstream.api.v1.consumer_list_re
 
 // JSApiConsumerGetNextRequest is for getting next messages for pull based consumers.
 type JSApiConsumerGetNextRequest struct {
-	Expires time.Time `json:"expires,omitempty"`
-	Batch   int       `json:"batch,omitempty"`
-	NoWait  bool      `json:"no_wait,omitempty"`
+	Expires time.Duration `json:"expires,omitempty"`
```

Comment:
fyi for Java client that this one would change /fyi @ColinSullivan1 @scottf

Reply:
@matthiashanel can we switch this change into a different branch? I think there is consensus on changing this one to duration so we can merge and integrate the change into the clients, but not sure right now on the side effects of changing some of the other ones.

Reply:
ok, will move the duration into a separate branch

Reply:
@wallyqs moved the duration change into https://github.com/nats-io/nats-server/pull/1944 and removed it from here